app/assets/javascripts/darkswarm/directives/open_street_map.js.coffee

```diff
@@ -11,34 +11,6 @@ Darkswarm.directive 'ofnOpenStreetMap', ($window, Enterprises, EnterpriseModal,
     openStreetMapProviderName = openStreetMapConfig.open_street_map_provider_name
     openStreetMapProviderOptions = JSON.parse(openStreetMapConfig.open_street_map_provider_options)
 
-    average = (values) ->
-      total = values.reduce (sum, value) ->
-        sum = sum + value
-      , 0
-      total / values.length
-
-    averageAngle = (angleName) ->
-      positiveAngles = []
-      negativeAngles = []
-      for enterprise in Enterprises.enterprises
-        if enterprise.latitude? && enterprise.longitude?
-          if enterprise[angleName] > 0
-            positiveAngles.push(enterprise[angleName])
-          else
-            negativeAngles.push(enterprise[angleName])
-
-      averageNegativeAngle = average(negativeAngles)
-      averagePositiveAngle = average(positiveAngles)
-
-      if negativeAngles.length == 0
-        averagePositiveAngle
-      else if positiveAngles.length == 0
-        averageNegativeAngle
-      else if averagePositiveAngle > averageNegativeAngle
-        averagePositiveAngle - averageNegativeAngle
-      else
-        averageNegativeAngle - averagePositiveAngle
-
     buildMarker = (enterprise, latlng, title) ->
       icon = L.icon
```

Comment:
Where's this `L` variable coming from? I can't see where it's set...?

Reply:
That `L` is the Leaflet JS library, it gets included here https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/assets/javascripts/darkswarm/all.js.coffee#L11